inference/utils/DanishFungiDataset.py

```diff
@@ -0,0 +1,138 @@
+import cv2
+import pandas as pd
+from albumentations import Compose, Normalize, Resize
+from albumentations.pytorch import ToTensorV2
+from torch.utils.data import Dataset
+
+
+class DanishFungiDataset(Dataset):
```

Comment:
why not use the fgvc dataset class since they need to install it anyway? It is nice for people who only want the dataset without training etc. thoug.